src/status_im2/contexts/chat/bottom_sheet_composer/effects.cljs

```diff
@@ -0,0 +1,91 @@
+(ns status-im2.contexts.chat.bottom-sheet-composer.effects
```

Comment:
@ilmotta Hmm, I see what you mean, but I disagree a little bit.

In the `handlers` file, I had the methods initially named as `handle-focus`, `handle-blur`, etc. But since I am calling these methods from another file, they are prepended with the namespace anyways, so it was `handle/handle-focus`, so it was redundant, so I changed it to `handle/focus`. Here these effects are called from the same file, so for example, having a method named `images` or `background` inside the `use-effect` call is a bit unclear I think.